### DISCUSSION TITLE
Manipulate event data in python

### DISCUSSION THREAD

**Discussion by allisonsampaio** [community/community#26834] [Actions]
<p>How can I send ${{github.event}} to a python file and manipulate that data with python?</p>

**Comment by ahdbilal** [comment, marked as answer]
<p>One way can be to pass this information via <code>env</code> keyword and then access it in python with <code>os.environ</code>.</p>
<p>In Action, set <code>github</code> as an environment variable:</p>
<pre><code class="lang-yaml">- name: pass env variable to python
  env: 
    GITHUB_CONTEXT: ${{ toJson(github) }}
  run: python test.py
</code></pre>
<p>In Python, you can access this with <code>os.environ</code></p>
<pre><code class="lang-python">import os
print(os.environ['GITHUB_CONTEXT'])
</code></pre>

**Comment by allisonsampaio** [comment]
<p>Thanks <a class="mention" href="/ahdbilal">@ahdbilal</a> , helped me a lot!</p>
<p>Do you know if there is any way to know the amount of comments opened by a user in an issue? And the number of issues opened by a user?</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/allisonsampaio">@allisonsampaio</a>,</p>
<blockquote>
<p>Do you know if there is any way to know the amount of comments opened by a user in an issue? And the number of issues opened by a user?</p>
</blockquote>
<p>You can try to use the “<a href="https://docs.github.com/en/free-pro-team@latest/rest/reference/issues#list-issue-comments" rel="noopener nofollow ugc"><strong>List issue comments</strong></a>” API to list all the comments on a specified issue.<br>
From the response of the API, you can see the details of each comment, include the comment body and the information of the user who added the comment.</p>
<p>You can traverse all the comments listed, and count the data you need.</p>
